src/api/votes.js

```diff
@@ -1,8 +1,21 @@
-import { VOTES } from '../mocks/votes'
+import { VOTES } from '../mocks/votes';
+
+let votes = [...VOTES];
 
 export const getVotes = () => {
+  const randomIndex = Math.floor(Math.random() * VOTES.length);
+  const randomVotes = Math.floor(Math.random() * 5) + 1;
+
+  const updatedVotes = [...votes];
+  updatedVotes[randomIndex] = {
```

Comment:
Why are we changing only one element per 'cycle'? This will not cause multiple elements to shift at the same time.